target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -319,6 +319,10 @@ tplink,tl-wpa8630p-v2-eu|\
 tplink,tl-wpa8630p-v2-int)
 	ucidef_set_led_switch "lan" "LAN" "tp-link:green:lan" "switch0" "0x3c"
 	;;
+tplink,tl-wpa8730-v1)
+	ucidef_set_led_wlan "wlan" "WLAN" "tp-link:green:wlan" "phy1tpt"
+	ucidef_set_led_wlan "wlan5" "WLAN5" "tp-link:green:wlan5" "phy0tpt"
+	;;
```

Comment:
This can be dropped entirely as it's in the DTS. The LED names are wrong anyway.

Reply:
dropped